examples/src/utils/renderer-debug-widget.ts

```diff
@@ -411,7 +411,7 @@ function updateWantedSectorOverride(
     const acceptedSimple = filterSectorNodes(quadsFilter, root);
     const acceptedDetailed = filterSectorNodes(detailedFilter, root);
     const wanted: reveal.WantedSector[] = [
-      ...cadNode.sectorScene
+      ...(cadNode.sectorScene as reveal.cadParsers.SectorScene)
```

Comment:
Why is this necessary now?

Reply:
Had problems compiling without it. May be that the problem is local. I'll change it and see if it works

Reply:
Update: It did not work. likely because the type of the argument variable in `map()` is not directly exposed through reveal. Although I don't see why that should be a problem :man_shrugging:

Reply:
Why are cadParsers exposed though?